ui/cypress.sh

```diff
@@ -1,16 +1,34 @@
 #!/usr/bin/env bash
 
+[[ "${IN_CI,,}" =~ ^(true)|(yes)|1$ ]] && IN_CI=true || IN_CI=''
+
 # On single node CONTROL_PLANE_IP = eth0
-CONTROL_PLANE_IP=$(ip a show eth0 | grep -Po "inet \K[\d.]+")
+CONTROL_PLANE_IP=$(/usr/sbin/ip a show eth0 | grep -Po "inet \K[\d.]+")
```

Comment:
This should be given as input to the script, there's no reason for us to assume it runs on a cluster member.

Reply:
True, didn't try to improve this script here, just wanted to adapt the stuff to make it work as before.